dind/cmd/cluster-up/main.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"time"
+
+	"github.com/golang/glog"
+	"k8s.io/test-infra/dind/pkg/cluster"
+	"k8s.io/test-infra/dind/pkg/util"
+)
+
+const (
+	masterConfigDir = "/var/kubernetes"
+	kubecfg         = "/var/kubernetes/admin.conf"
+	timeout         = time.Duration(5) * time.Minute
+)
+
+var (
```

Comment:
Would be nice to follow the `type options struct{}` convention we have in `prow`

Reply:
I don't want to start creating a monolithic config struct yet, because I suspect the configuration might soon evolve rapidly.

For instance, do we want to allow a kubeadmin config to be passed through, or do we want to wrap that functionality with something at a higher level?